dashboard/test/controllers/api_controller_test.rb

```diff
@@ -112,6 +135,59 @@ def make_progress_in_section(script)
     assert_equal expected_response, JSON.parse(@response.body)
   end
 
+  test "should get assessments for section with script with level_group assessment" do
+    script = create :script
+
+    sub_level1 = create :text_match, name: 'level_free_response', type: 'TextMatch'
+    sub_level2 = create :multi, name: 'level_multi_unsubmitted', type: 'Multi'
+    sub_level3 = create :multi, name: 'level_multi_correct', type: 'Multi'
+    sub_level4 = create :multi, name: 'level_multi_incorrect', type: 'Multi'
+
+    # create 2 level_group levels
+    level1 = create :level_group, name: 'LevelGroupLevel1', type: 'LevelGroup'
+    level1.properties['title'] =  'Long assessment 1'
+    level1.properties['pages'] = [{levels: ['level_free_response', 'level_multi_unsubmitted']}, {levels: ['level_multi_correct', 'level_multi_incorrect']}]
+    level1.save!
+    create :script_level, script: script, level: level1, assessment: true
+
+    # student_1 has an assessment
+    create(:activity, user: @student_1, level: level1,
+           level_source: create(:level_source, level: level1,
+            data: "[{\"level_id\": #{sub_level1.id},\"result\":\"This is a free response\"},{\"level_id\": #{sub_level2.id},\"result\":\"0\"},{\"level_id\": #{sub_level3.id},\"result\":\"1\"},{\"level_id\":#{sub_level4.id},\"result\":\"-1\"}]"
+            ))
+
+    updated_at = Time.now
+
+    create :user_level, user: @student_1, best_result: 100, script: script, level: level1, submitted: true, updated_at: updated_at
+
+    get :section_assessments, section_id: @section.id, script_id: script.id
+    assert_response :success
+
+    assert_equal script, assigns(:script)
+
+    # all these are translation missing because we don't actually generate i18n files in tests
+
+    expected_response =
+      [
+       {"student"=>{"id"=>@student_1.id, "name"=>@student_1.name},
+         "stage"=>"translation missing: en-us.data.script.name.bogus_script_1.title",
```

Comment:
I'm not quite sure how the previous test (for section_text_responses) gets this stage name with extra words: "Stage 1: translation missing: en-us.data.script.name.#{script.name}.#{script.stages[0].name}"